### ISSUE TITLE
Removing shapes created by Geoman Draw

### ISSUE THREAD

**Issue by justinjngan** [geoman-io/leaflet-geoman#1067]
Hello 

First off thank you to Geoman and supporters for creating an awesome replacement for leaflet draw.  Leaflet draw is good and true to its minimalist approach. However, snapping arguably is critical to ensure data quality.

I am using Geoman as a creation interface. I listen for pm:create and customize handling which includes creating a geojson representation of what was created along with attributes that the user supplies in a modal. 

Right now, it works but of course, in addition to the geojson object with attributes that I add to a L.GeoJSON layer, there is also the shape created by Geoman draw.  

I'm wondering, how can I remove or prevent Geoman from actually creating on the map, the layer that is created?

MANY THANKS!

**Comment by Falke-Design** [comment]
I think you search for the layerGroup option:
```
map.pm.setGlobalOptions({ layerGroup: YOUR_LAYER_GROUP });
```

If you set this option all layers will be created on that LayerGroup / FeatureGroup / GeoJSON layer.

**Comment by justinjngan** [comment]
Ah, Falke-Design, I've read the kudos to you for all your support. Well, thank you!

I will try that. Right now, I didn't even initialize any global options!  This will take a bit longer as I have to switch to another project but I will come back and log the result.

Out of curiosity, is there any interest to update some of the Readme ?  There's at least one section that doesn't work as suggested. This is:
`map.pm.addControls({  
  position: 'topleft',  
  drawCircle: false,  
});  `

It appears that some other options are required:
`		map.pm.addControls({ editMode: true, 
			oneBlock: true, 
			cutPolygon: false,
			drawMarker: false, 
			drawCircleMarker: false,
			drawCircle: false,
			drawPolyline: false,
			drawPolygon: false, 
			drawRectangle: false,
			editMode: false,
			dragMode: false,
			removalMode: false,
			rotateMode: false });`

Specifically, I think it is editMode: true.

**Comment by Falke-Design** [comment]
>  There's at least one section that doesn't work as suggested. This is:

What is not working? What do you expect? If you set `drawCircle: false` the Circle Button will be removed from the Toolbar and that is working

**Comment by justinjngan** [comment]
Well, absolutely everything is fine now, which is perplexing as all morning I was working on getting Geoman draw controls to load on the map. I used a bare leaflet map and added the .css and .js files.  I then tried to add the draw controls but I received an Uncaught Type Error: n is undefined. This error appeared to go away as I added more options to  pm.addControls. 

However, in trying to replicate the error, by removing everything except for the .css/.js and the pm.addControls lines, I no longer received the error.
